src/bus_monitor.cpp

```diff
@@ -670,129 +670,67 @@ void SystemStatus::listenBootProgressState()
         });
 }
 
-void SystemStatus::powerPolicyStateCallback(sdbusplus::message::message& msg)
+void SystemStatus::biosAttribsCallback(sdbusplus::message::message& msg)
 {
-    std::string object{};
-    types::ItemInterfaceMap invItemMap;
-
-    msg.read(object, invItemMap);
-    const auto itr = invItemMap.find("PowerRestorePolicy");
-    if (itr != invItemMap.end())
+    if (msg.is_method_error())
     {
-        if (auto powerState = std::get_if<std::string>(&(itr->second)))
-        {
-            // std::cout << "power state = " << *powerState << std::endl;
-            powerPolicy = *powerState;
-
-            setSystemCurrentOperatingMode();
-        }
-        else
-        {
-            std::cerr << "Failed to read power policy from Dbus" << std::endl;
-        }
+        std::cerr << "Error in reading BIOS attribute signal " << std::endl;
```

Comment:
correct me if I m understanding it wrong.
IS this-  BIOS attribute value?
OR - BIOS attribute changed signal

Reply:
The callback is outcome of signal received.